introduction/edge-functions.mdx

```diff
@@ -0,0 +1,59 @@
+# Edge functions
+Edge Functions are server-side functions that run directly within your database environment. Edge functions in SQLite Cloud ensure maximum performance and minimal latency by running functions on the same node as your database.
+
+You can write edge functions directly in the SQLite Cloud dashboard using JavaScript, TypeScript, SQL, or Python (experimental). You can [import modules](https://bun.sh/docs/runtime/autoimport) into your JavaScript and TypeScript edge functions. Module imports for Python are not currently supported.
```

Comment:
At the moment `Import` isn't supported. 
To support bun auto-install we have to change how javascript and typescript code is executed.